### DISCUSSION TITLE
[Feature Request] Publish source maps to NPM

### DISCUSSION THREAD

**Discussion by christopherdro** [alexandercerutti/passkit-generator#182] [Ideas]
It seems that source maps are included in the builds but not published to NPM. Is there a specific reason for their exclusion? If not, could we request a new release with them included?

**Comment by alexandercerutti** [comment]
Hi @christopherdro! Thanks for using `passkit-generator`!
I'm used to not including them in the final output, but it might be reasonable to include them.

Do you have any specific reason why you might need it?

**Reply by christopherdro** [reply]
The project where this library is used requires a source map for local development. As a result, I receive multiple warnings indicating that the source map file is missing.

>Failed to parse source map from '/node_modules/passkit-generator/lib/schemas/Barcode.js.map' file: Error: ENOENT: no such file or directory, open '/node_modules/passkit-generator/lib/schemas/Barcode.js.map'

**Reply by alexandercerutti** [reply]
I see. Warnings are just warnings so they are not a problem.
So, if I understand correctly, the solutions here are two:

 - Not emit at all source maps when building for distribution
 - Publishing them.

Am I right?

**Reply by christopherdro** [reply]
> I see. Warnings are just warnings so they are not a problem.

Yes, it's more of an eyesore than a problem, and with live reload on code changes, my terminal gets flooded with these warnings. 

> So, if I understand correctly, the solutions here are two:
Not emit at all source maps when building for distribution
Publishing them.

I believe so. However, I think I've identified an issue in the webpack configuration of this project where they attempt to suppress source map warnings. I've just fixed it, so it's up to you whether you want to include source maps as part of the release.

**Reply by alexandercerutti** [reply]
I'll keep it in consideration for the next release, thanks for bringing it up!

**Reply by alexandercerutti** [reply]
v3.2.0 is going to include inline source maps.
Sorry it took this long, but no big maintenance is required in this package when there are no news to integrate :)
